coral-hive/src/main/java/com/linkedin/coral/hive/hive2rel/functions/VersionedSqlUserDefinedFunction.java

```diff
@@ -68,6 +87,27 @@ public String getViewDependentFunctionName() {
     return viewDependentFunctionName;
   }
 
+  /**
+   * Retrieves the short function name based on the class name. If the class name is found
+   * in the predefined {@link VersionedSqlUserDefinedFunction#SHORT_FUNC_NAME_MAP},
+   * the corresponding short name is returned. Otherwise, the method converts the last
+   * segment of the class name from UPPER_CAMEL to LOWER_UNDERSCORE format to generate
+   * the short function name.
+   */
+  public String getShortFunctionName() {
+    String unversionedClassName = getName();
+    if (SHORT_FUNC_NAME_MAP.containsKey(unversionedClassName)) {
+      return SHORT_FUNC_NAME_MAP.get(unversionedClassName);
+    }
+    Converter<String, String> caseConverter = CaseFormat.UPPER_CAMEL.converterTo(CaseFormat.LOWER_UNDERSCORE);
+    String[] nameSplit = unversionedClassName.split("\\.");
+    return caseConverter.convert(nameSplit[nameSplit.length - 1]);
+  }
+
+  public String getUDFClassName() {
+    return udfClassName;
+  }
```

Comment:
FunctionName vs UDF Name.. can we fix all terminology and unify them to minimize confusion?

Reply:
Modified